src/browser/focus.js

```diff
@@ -1,6 +1,12 @@
-import { sendKeys } from '@web/test-runner-commands';
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
 
-export const focusWithKeyboard = async(element) => {
```

Comment:
Does linting want the `function` setup versus the `const` setup, or just trying to match the style of the repo? Or is there an advantage of one over the other?

Reply:
This isn't linting related, it was more of a personal preference. TBH I'm not sure where or why the `const` setup was introduced... _maybe_ [in the old visual-diff library](https://github.com/BrightspaceUI/visual-diff/blob/main/helpers/focus.js#L12) and it got copied a bunch of places? If someone knows more about why we'd want one vs. the other, chime in!

Reply:
I'm not sure, I feel like we use the `const` style randomly throughout in `core`, but it's not consistent. https://github.com/BrightspaceUI/core/blob/main/components/form/form-helper.js
https://github.com/BrightspaceUI/core/blob/main/helpers/gestures.js
And I just flip back and forth depending on what the file is already doing, so was mostly curious!